bears/general/InvalidLinkBear.py

```diff
@@ -72,12 +81,20 @@ def run(self, filename, file,
         `do_not_ever_open = 'https://api.acme.inc/delete-all-data'` wiping out
         all your data.
 
-        :param timeout:          Request timeout period.
+        :param network_timeout:       A dict containing the url and the timeout
         :param link_ignore_regex:     A regex for urls to ignore.
         :param follow_redirects: Set to true to autocorrect redirects.
         """
+        host_dict = dict()
```

Comment:
yeah.. I can.. will do it by tomorrow evening IST/GMT +5:30 

It didn't get merged by tonight! :(